README.md

```diff
@@ -402,22 +475,64 @@ Example `process_exec` and `process_exit` events can be:
 </p>
 </details>
 
-For the rest of the use cases we will use the Tetragon CLI to give the output.
+#### Use case 2: Privileged Execution
+
+Tetragon also provides the ability to check process capabilities and kernel namespaces.
+
+As a first step let's enable visibility to capability and namespace changes via the configmap
+by setting `enable-process-cred` and `enable-process-ns`  from `false` to `true`
+```bash
+kubectl edit cm -n kube-system tetragon-config
+# change "enable-process-cred" from "false" to "true"
+# change "enable-process-ns" from "false" to "true"
+# then save and exit
+```
+Restart the Tetragon daemonset:
+```
+kubectl rollout restart -n kube-system ds/tetragon
+```
+
+As a second step, let's start monitoring the Security Observability events from the privileged `test-pod` workload:
+```bash
+kubectl logs -n kube-system -l app.kubernetes.io/name=tetragon -c export-stdout -f | tetra getevents --namespace default --pod test-pod
+```
+
+In another terminal let's apply the privileged PodSpec:
+```bash
+kubectl apply -f https://raw.githubusercontent.com/cilium/tetragon/main/testdata/specs/testpod.yaml
+```
+
+If you observe the output in the first terminal, you can see the container start with `CAP_SYS_ADMIN`:
+```bash
+🚀 process default/test-pod /bin/sleep 365d                🛑 CAP_SYS_ADMIN
+🚀 process default/test-pod /usr/bin/jq -r .bundle         🛑 CAP_SYS_ADMIN
+🚀 process default/test-pod /usr/bin/cp /kind/product_name /kind/product_uuid /run/containerd/io.containerd.runtime.v2.task/k8s.io/7c7e513cd4d506417bc9d97dd9af670d94d9e84161c8c8 fdc9fa3a678289a59/rootfs/ 🛑 CAP_SYS_ADMIN
+```
+
+
+### Generic tracing
+
+For more advanced use cases, Tetragon can observe tracepoints and arbitrary
+kernel calls via kprobes. For that, Tetragon must be extended and configured
+with custom resources objects named `TracingPolicy`. It can then generates
+`process_tracepoint` and `process_kprobes` events.
 
-### File Access
+`TracingPolicy` is a user-configurable Kubernetes custom resource that allows
+users to trace arbitrary events in the kernel and optionally define actions to
+take on a match, for enforcement for example. For bare metal or VM use cases
```

Comment:
.. take on a match. For example, in case all the selectors are matching in the TracingPolicy, we can send a Sigkill signal to the process or override the return value of the system call.